packages/preview/glossarium-0.1.0/README.md

```diff
@@ -0,0 +1,99 @@
+# Typst glossary
+
+> Glossarium is based in great part of the work of [Sébastien d'Herbais de Thun](https://github.com/Dherse) from his master thesis available at: <https://github.com/Dherse/masterproef>. His glossary is available under the MIT licence [here](https://github.com/Dherse/masterproef/blob/main/elems/acronyms.typ).
+
+Glossarium is a simple, easily customizable typst glossary inspired by [LaTeX glossaries package](https://www.ctan.org/pkg/glossaries) . You can see an example showing the different features in [`example.typ`](example/example.typ).
+
+![Screenshot](example/example.png)
+
+## Manual 
+
+### Import and setup
+
+This manual assume you have a good enough understanding of typst markup and scripting. 
+
+For Typst 0.6.0 or later import the package from the typst preview repository:
+
+```ts
+#import "@preview/glossarium:0.1.0": make-glossary, print-glossary, gls, glspl 
+```
+
+For Typst before 0.6.0 or to use **glossarium** as a local module, download the package files into your project folder and import `glossarium.typ`:
+
+```js
+#import "glossarium.typ": make-glossary, print-glossary, gls, glspl 
+```
+
+After importing the package and before making any calls to `gls`,` print-glossary` or `glspl`, please ***MAKE SURE*** you add this line
+```js
+#show: make-glossary.with()
```

Comment:
The `.with()` shouldn't have any effect without arguments.
```suggestion
#show: make-glossary
```

Reply:
changed